test/PreAMMBatcher-e2e.ts

```diff
@@ -160,28 +160,28 @@ describe("PreAMMBatcher-e2e", () => {
         [walletTrader1, walletTrader2],
         [walletTrader3, walletTrader4],
       ),
-      true,
+      false,
     );
-    console.log(testCase.sellOrdersToken0.length);
-    console.log(testCase.sellOrdersToken0[0].sellToken.address);
+    // console.log(testCase.sellOrdersToken0.length);
+    // console.log(testCase.sellOrdersToken0[0].sellToken.address);
     expect(testCase.solution.sellOrdersToken0.length).to.be.equal(1);
     expect(testCase.solution.sellOrdersToken1.length).to.be.equal(1);
     await runScenarioOnchain(testCase);
 
-    console.log(
-      "auction clearing price:",
-      testCase.solution.clearingPrice.numerator
-        .mul(BigNumber.from("100000"))
-        .div(testCase.solution.clearingPrice.denominator)
-        .toString(),
-    );
-    console.log(
-      "uniswap clearing price:",
-      (await uniswapPair.getReserves())[0]
-        .mul(100000)
-        .div((await uniswapPair.getReserves())[1])
-        .toString(),
-    );
+    // console.log(
+    //   "auction clearing price:",
+    //   testCase.solution.clearingPrice.numerator
+    //     .mul(BigNumber.from("100000"))
+    //     .div(testCase.solution.clearingPrice.denominator)
+    //     .toString(),
+    // );
+    // console.log(
+    //   "uniswap clearing price:",
+    //   (await uniswapPair.getReserves())[0]
+    //     .mul(100000)
+    //     .div((await uniswapPair.getReserves())[1])
+    //     .toString(),
+    // );
```

Comment:
I'm not a fan of commented out "dead" code, there is nothing that verifies that it will continue to run after a refactor when uncommented. I suggest using  <https://www.npmjs.com/package/debug> package. This is a thin `console.log` wrapper that is controlled by environment varibles. So `yarn test` would not log to console at all and `DEBUG=* yarn test` would. That way there is no need to comment and uncomment code when debugging, and the commented code doesn't bit-rot.